pricefeed/pricefeed_test.go

```diff
@@ -17,16 +18,21 @@ func init() {
 	logging.SetLogLevel("pricefeed", "debug")
 }
 
-func getMainnetEndpoint() string {
+func getMainnetEndpoint(t *testing.T) string {
 	endpoint := os.Getenv("ETH_MAINNET_ENDPOINT")
 	if endpoint == "" {
 		endpoint = mainnetEndpoint
 	}
+	eURL, err := url.Parse(endpoint)
+	require.NoError(t, err)
+	// path and fragments may have API keys, so don't log them
+	t.Logf("mainnet endpoint is %s://%s", eURL.Scheme, eURL.Host)
```

Comment:
Interestingly, CI redacted this even though the secret part of the secret was omitted:
```
pricefeed_test.go:29: mainnet endpoint is ***
```
Due to the fact that it was redacted, it looks like we are using the correct value, so the log isn't useless.